submissions/description/cross_site_scripting_xss/ie_only/older_version_ie_eleven/template.md

```diff
@@ -0,0 +1,30 @@
+# Cross-Site Scripting (Internet Explorer Older Than 11)
+
+## Overview of the Vulnerability
+
+Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. XSS can be found on this domain which allows an attacker to control code that is executed within a user’s Internet Explorer browser which is older than version 11.
```

Comment:
A small improvement to flow

Cross-Site Scripting (XSS) is a type of injection attack where malicious JavaScript is injected into a website. When a user visits the affected web page, the Javascript executes within that user’s browser in the context of the domain. This XSS can be found on the domain listed, which allows an attacker to insert code that is executed within the user's browser, as long as it is Internet Explorer prior to Version 11.